dev/projection/animate-single-element-layout-change-with-child.html

```diff
@@ -85,23 +85,25 @@
 
             boxProjection.root.didUpdate()
 
-            requestAnimationFrame(() => {
-                matchViewportBox(box, {
-                    top: 50,
-                    right: 270,
-                    bottom: 170,
-                    left: 100,
+            sync.postRender(() => {
+                sync.postRender(() => {
```

Comment:
By nesting these two frames deep is there not a flash of the wrong bounding box?

Reply:
The first frame would be still on progress 0 and the second to the mid box right?

Reply:
Oh yeah doh